CMakeLists.txt

```
@@ -3,9 +3,17 @@ project(3DMMForever LANGUAGES CXX C VERSION 0.2)
 
 list(APPEND CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake")
 
+set(CMAKE_CONFIGURATION_TYPES "Debug" "Release")
+
 set(CMAKE_MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>")
 set(CMAKE_CXX_STANDARD 14)
 
+# Generate PDBs for release builds
+if(MSVC)
+    set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} /Zi")
+    set(CMAKE_EXE_LINKER_FLAGS_RELEASE "${CMAKE_SHARED_LINKER_FLAGS_RELEASE} /debug /OPT:REF")
+endif()
```

I'd recommend doing

```suggestion
add_compile_options($<AND:$<CONFIG:Release>,$<COMPILE_LANG_AND_ID:CXX,MSVC>>:/Zi>)
add_link_options($<AND:$<CONFIG:Release>,$<LINK_LANG_AND_ID:CXX,MSVC>>:/OPT:REF>)
```

I'd recommend NOT adding `/debug` to a release build. You can add a preset that uses `CMAKE_BUILD_TYPE=RelWithDebInfo` instead.

---

I changed the Release config flags because the RelWithDebInfo config didn't build. I'll revert these changes and get the RelWithDebInfo config building instead.